eng/Version.Details.xml

```diff
@@ -293,87 +293,95 @@
       <Uri>https://github.com/aspnet/Extensions</Uri>
       <Sha>3e7f99fbe3c5e775f5c175238eed87523782fe13</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CSharp" Version="4.6.0-preview7.19309.5" CoherentParentDependency="Microsoft.NETCore.App">
+    <Dependency Name="Microsoft.CSharp" Version="4.6.0-preview7.19309.5" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
Suggest Microsoft.NETCore.App.Ref as the `CoherentParentDependency` value for everything coming from CoreFx. It's shorter and more obviously applicable everywhere.

Reply:
I'm not sure if that would work. Eventually, the targeting pack (.Ref) would stop revving after a patch is released, which means this dependency wouldn't get updated. @natemcmaster to confirm?

Reply:
:ok: it was a suggestion and you're right that it could stop working. Your current changes will work, just looked a little odd.